screenpy_playwright/protocols.py

```diff
@@ -7,3 +7,6 @@ class PageObject(Protocol):
     """PageObjects have a URL."""
 
     url: str
+
+
+__all__ = ["PageObject"]
```

Comment:
This one was odd because we do `from .protocols import *` in `__init__.py`.  I figured we could either do individual imports in the `__init__` or just add an `__all__` to `protocols.py`.  

This felt like less work in the future should any new protocols get added. /shrug

Reply:
Hm... i'm kinda surprised it works. I don't know why it wouldn't, i just never thought to try putting `__all__` in a non-`__init__` file.

It doesn't follow the pattern we've got on the [other projects](https://github.com/ScreenPyHQ/screenpy/blob/trunk/screenpy/__init__.py#L46) though. I think it's weirder to have an `__all__` in here than it is to import each piece in `__init__.py`. 

Though... i guess i don't know how common it is to put an `__all__` in a file like this. Maybe this is the standard we should be following in the other projects?

Reply:
I can go either way, tbh.  I'm not sure I have a preference.

Reply:
I updated it to follow the pattern.